FastMenu/README.md

```diff
@@ -4,6 +4,20 @@ Several patches to speed up the loading of the main settings menu:
 - Remove another animation for fading in the menu contents (why do they have two).
 - Eagerly load the menu contents; without this the first time has an extra delay.
 
+## Requirements
```

Comment:
This makes it look like this is requirements for using the plugin, which is untrue; this is only needed to satisfy TS.

Reply:
Forms.FocusLock just doesn't exist which are needed in the file so technically speaking it is a requirement for the plugin to work.

Reply:
It is not a requirement for the plugin to work. It's a requirement for it to pass typescript.